### DISCUSSION TITLE
ToA Warden Insanity invocation

### DISCUSSION THREAD

**Discussion by Jetsyy** [runelite/runelite#15655] [Q&A] [answered]
Could we get a plug-in to track the floor attack on Warden's phase 3 with insanity invocation? If you're not aware, for phase 3 you have to step on the right side of the floor, then the left, then the middle. It goes in that order every time. Then at certain hp thresholds he summons skulls you have to kill. Without the insanity invocation, it always starts at the left side. With the insanity invocation, the pattern picks up where it left off previously. I'd like a plugin that can keep track of his floor attacks so you can know where he left off after the skull attack.

**Comment by Enriath** [comment, marked as answer]
Banned by the [Jagex Statement](https://secure.runescape.com/m=news/another-message-about-unofficial-clients?oldschool=1):
* Next attack prediction
* Anything that automatically indicates where to stand, or not to stand
* Adds additional visual or audio indicators of a boss mechanic
